examples/inverse/time_frequency_mixed_norm_inverse.py

```diff
@@ -50,7 +50,6 @@
 # Handling average file
 condition = 'Left visual'
 evoked = mne.read_evokeds(ave_fname, condition=condition, baseline=(None, 0))
-evoked = mne.pick_channels_evoked(evoked)
```

Comment:
presumably this was there to restrict to data channels? a quick look at following code suggests it won't matter, but we should eyeball the resulting plots to make sure.

Reply:
Checked by eye, all good!